### PR TITLE
Release cards 2 11

### PR DESCRIPTION
**Pull request by BenWilson2**
<details><summary>&#x1F6E0 DevTools &#x1F6E0</summary>
<p>

[Open in GitHub Codespaces](https://codespaces.new/BenWilson2/mlflow/pull/11289?quickstart=1)

#### Install mlflow from this PR

```
pip install git+https://github.com/mlflow/mlflow.git@refs/pull/11289/merge
```

#### Checkout with GitHub CLI

```
gh pr checkout 11289
```

</p>
</details>

### Related Issues/PRs

 #xxx

### How is this PR tested?

- [ ] Existing unit/integration tests
- [ ] New unit/integration tests
- [X] Manual tests

### Does this PR require documentation update?

- [ ] No. You can skip the rest of this section.
- [X] Yes. I've updated:
  - [ ] Examples
  - [ ] API references
  - [X] Instructions

#### Is this a user-facing change?

- [X] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

#### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/deployments`: MLflow Deployments client APIs, server, and third-party Deployments integrations
- [X] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>

#### How should the PR be classified in the release notes? Choose one:

- [X] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: docs/source/new-features/index.rst [hunk 1]
```diff
@@ -46,6 +67,82 @@ Find out about the details of major features, changes, and deprecations below.
...
+                    </div>
+                    <div class="tag">
+                        <a href="https://github.com/mlflow/mlflow/releases/tag/v2.11.0">released in 2.11.0</a>
+                    </div>
+                </div>
+            </div>
+            <div class="grid-card">
+                <div class="content-container">
+                    <div class="header">
+                        Automated model checkpointing support added for Deep Learning model training
```
Comment (B-Step62) [inline docs/source/new-features/index.rst:89]:
```suggestion
                        Automated model checkpointing for Deep Learning model training
```

super-nit: To make the title renders within 2 rows in default and max page width (all other cards do)

File: docs/source/new-features/index.rst [hunk 2]
```diff
@@ -46,6 +67,82 @@ Find out about the details of major features, changes, and deprecations below.
...
+                </div>
+            </div>
+            <div class="grid-card">
+                <div class="content-container">
+                    <div class="header">
+                        MLflow now has support for OpenAI SDK 1.x
+                    </div>
+                    <img class="card-image" src="../_static/images/logos/openai-logo.svg" alt="OpenAI" style="max-height: 5rem"></img>
+                    <div class="body">
+                        We've updated flavors that interact with the OpenAI SDK, bringing full support for the API changes with the 1.x release.
```
Comment (B-Step62) [inline docs/source/new-features/index.rst:139]: Do we have any good reference for the required code update for users? I guess it's only `task=openai.ChatCompletion` -> `task=openai.chat.Completion` and we've updated code examples in our doc ([here](https://github.com/mlflow/mlflow/pull/11123/files#diff-27092883ef9a4a81ab9b5ea59442ab48dd314babd61428270948fcfae0a80f28)). Maybe we can add a link to one of them as quick reference?

File: docs/source/new-features/index.rst [hunk 3]
```diff
@@ -46,6 +67,82 @@ Find out about the details of major features, changes, and deprecations below.
...
+            </div>
+            <div class="grid-card">
+                <div class="content-container">
+                    <div class="header">
+                        Automated model checkpointing support added for Deep Learning model training
+                    </div>
+                    <img class="card-image" src="../_static/images/logos/mlflow-logo.svg" alt="MLflow"></img>
+                    <div class="body">
+                        When performing training of Deep Learning models with PyTorch Lightning or Keras, model checkpoint saving 
+                        is enabled, allowing for state storage during long-running training events and the ability to resume if
```
Comment (B-Step62) [inline docs/source/new-features/index.rst:94]: nit: Do we have a doc reference for how to enabling and configuring the checkpointing?

**Review by harupy — APPROVED** [review]
LGTM!

File: docs/source/new-features/index.rst [hunk 4]
```diff
@@ -9,14 +9,35 @@ Find out about the details of major features, changes, and deprecations below.
...
             <div class="grid-card">
                 <div class="content-container">
                     <div class="header">
                         PEFT (Parameter-Efficient Fine-Tuning) support
                     </div>
                     <img class="card-image" src="../_static/images/logos/huggingface-logo.svg" alt="HuggingFace Logo"></img>
                     <div class="body">
-                        MLflow now natively supports PEFT (Parameter-Efficient Fine-Tuning) models in Transformers flavor. PEFT unlocks significantly efficient model fine-tuning like LoRA, QLoRA, Prompt Tuning. Check it out to build your own cutting-edge models with MLflow and PEFT!
+                        MLflow now natively supports PEFT (Parameter-Efficient Fine-Tuning) models in the Transformers flavor. PEFT 
+                        unlocks significantly efficient model fine-tuning like LoRA, QLoRA, and Prompt Tuning. Check it out to
```
Comment (B-Step62) [inline docs/source/new-features/index.rst:39]:
```suggestion
                        unlocks significantly efficient model fine-tuning like LoRA, QLoRA, and Prompt Tuning. Check out `the new QLoRA fine-tuning tutorial <../llms/transformers/tutorials/fine-tuning/transformers-peft.html>`_ to 
```

Self-update: "Learn more" doesn't stand out so adding in-line link.

**Review by B-Step62 — APPROVED** [review]
LGTM overall! Just a few minor comments about references for each feature.

File: docs/source/new-features/index.rst [hunk 5]
```diff
@@ -9,14 +9,35 @@ Find out about the details of major features, changes, and deprecations below.
...
+            </div>
             <div class="grid-card">
                 <div class="content-container">
                     <div class="header">
                         PEFT (Parameter-Efficient Fine-Tuning) support
                     </div>
                     <img class="card-image" src="../_static/images/logos/huggingface-logo.svg" alt="HuggingFace Logo"></img>
                     <div class="body">
-                        MLflow now natively supports PEFT (Parameter-Efficient Fine-Tuning) models in Transformers flavor. PEFT unlocks significantly efficient model fine-tuning like LoRA, QLoRA, Prompt Tuning. Check it out to build your own cutting-edge models with MLflow and PEFT!
+                        MLflow now natively supports PEFT (Parameter-Efficient Fine-Tuning) models in the Transformers flavor. PEFT
```
Comment (B-Step62) [inline docs/source/new-features/index.rst:38]:
```suggestion
                        MLflow now natively supports `PEFT (Parameter-Efficient Fine-Tuning) <../llms/transformers/guide/index.html#peft-models-in-mlflow-transformers-flavor>`_ models in the Transformers flavor. PEFT 
```

self-update: same as the below comment

File: docs/source/new-features/index.rst (hunk 1 above)
Comment (BenWilson2) [reply docs/source/new-features/index.rst:89]: not a nit at all. Great idea :D

File: docs/source/new-features/index.rst (hunk 3 above)
Comment (BenWilson2) [reply docs/source/new-features/index.rst:96]: Good thing you pointed this out - we were missing the docstrings for tensorflow! Added those and updated with links to the API docs

**Pull request merged**